models/protected-degradation.jl

```diff
@@ -146,12 +158,17 @@ function rates( states::Dict, parameters::Dict, t::Float64 )
 
 		dc₆[end] = D₆ * (c₆[end-1]-c₆[end]) / Δx^2
 		dc₁₂[end] = D₁₂ * (c₁₂[end-1]-c₁₂[end]) / Δx^2
+
+		# relay reaction
+		dc₆ .+= ρ .* k₆ .* luxI
+		dc₁₂ .+= ρ .* k₁₂ .* lasI
```

Comment:
@luca-rosa if you want to change the functional form of the relay reaction do this here